pageserver/src/config.rs

```diff
@@ -560,6 +576,7 @@ impl PageServerConf {
                     let permits = NonZeroUsize::new(permits).context("initial semaphore permits out of range: 0, use other configuration to disable a feature")?;
                     ConfigurableSemaphore::new(permits)
                 }),
+                "testing_mode" => builder.testing_mode(parse_toml_bool(key, item)?),
```

Comment:
`if let Some(trace_read_requests) = item.get("trace_read_requests") {` also uses bool parsing, let's put that method in there too.